src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/takemanagement/viewmodel/TakeManagementViewModel.kt

```diff
@@ -31,6 +32,9 @@ class TakeManagementViewModel : ViewModel() {
     private val takeRepository = injector.takeRepository
     private val pluginRepository = injector.pluginRepository
 
+    private val takesViewModel: TakesViewModel by inject()
+    private val workbookViewModel: WorkbookViewModel by inject()
```

Comment:
These are added so that we can call the new record() function (line 160.) However, TakeManagementViewModel and TakesViewModel will be merged soon, so the changes to this file will be obsolete when that happens.

Reply:
Update 6/26: I removed takesViewModel from here. TakesViewModel actually got renamed to RecordResourceViewModel, which does not contain any recording business logic. TakeManagementViewModel is now the thing that houses logic for recording/editing/etc Takes (and a lot more). 

Likewise, I think we should have a RecordScriptureViewModel that only contains things needed to bind data to the "scripture takes" view. The current TakeManagementViewModel probably needs to be greatly refactored.